lib/helper.js

```diff
@@ -37,9 +38,23 @@ let logofied = text => {
  */
 let logger = (data, error) => {
   if (error) {
-    console.log(chalk.red(`${JSON.stringify(data, null, 2)}`));
+    let exclamationMarks =
```

Comment:
What does this mean?

Reply:
Those are the changes I made to complete my previous task, add emoji to the output. For this task, I made a few modification in bin/nc.js.

Reply:
Hi @duc0905 a PR should contain code related to a particular task only. can you remove unrelated code from this task, please?